app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java

```diff
@@ -42,13 +42,25 @@ protected void onCreate(Bundle savedInstanceState) {
 
     private void checkIfLoggedIn() {
         boolean islogin = sharedPreferences.getBoolean("isLogin", false);
-        if(!islogin) {
+        if (!islogin) {
             Intent i = new Intent(this, LoginActivity.class);
             startActivity(i);
             finish();
         }
     }
 
+    public void logOut() {
+        SharedPreferences.Editor editor = sharedPreferences.edit();
+            editor.putString("userName", "");
+            editor.putString("password", "");
+            editor.apply();
```

Comment:
@zsofinnemeth almost :) these 3 lines are not in a new block, so they should be indented as the same as the above editor creation. Also resolve conflicts and then this PR is mergable.